### PR TITLE
one js file

### PR DESCRIPTION
**Pull request by redwing555**
this PR is a wat of enabling my coding partner to review my code for JS best practices

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  }));
+}
+
+
+function addTaskToStorage() {
+  const tasksInfo = JSON.stringify(tasks);
+  localStorage.setItem('tasksList', tasksInfo);
+}
+
+function AddTask() {
```
Comment (mahbubul14) [inline main.js:31]: - [ ]  Kindly change all regular functions to es6 arrow function syntax following the JavaScript best practices guide.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  const task = {
+    description: document.querySelector('.input-task').value,
+    completed: false,
+    index: tasks.length,
+  };
```
Comment (mahbubul14) [inline main.js:36]: - [ ]  Kindly change all regular functions to es6 arrow function syntax following the JavaScript best practices guide.

Comment (mahbubul14) [inline main.js]: - [ ]  Kindly change all regular functions to es6 arrow function syntax following the JavaScript best practices guide.

Comment (redwing555) [reply main.js]: Thank you for the code review it was very helpful , happy coding  👍 😃

**Pull request merged**